### PR TITLE
Add version streams 2.7 and 2.8 for argo-cd

### PR DESCRIPTION
**Pull request by tcnghia**
* "argo-cd-2.7 & argo-cd-2.8: add version streams"

### CODE REVIEW

File: argo-cd-2.8.yaml [hunk 1]
```diff
@@ -100,9 +105,13 @@ subpackages:
           ln -s /usr/local/bin/argocd "${{targets.subpkgdir}}"/usr/local/bin/argocd-notifications
           ln -s /usr/local/bin/argocd "${{targets.subpkgdir}}"/usr/local/bin/argocd-applicationset-controller
           ln -s /usr/local/bin/argocd "${{targets.subpkgdir}}"/usr/local/bin/argocd-k8s-auth
+    dependencies:
```
Comment (mattmoor) [inline argo-cd-2.8.yaml:108]: I believe you have to rename the subbpackage names

File: argo-cd-2.7.yaml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        - openssh-server
+      provides:
+        - argo-cd-repo-server=2.7.999
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/bin
+          cp hack/gpg-wrapper.sh "${{targets.subpkgdir}}"/usr/bin/gpg-wrapper.sh
+          cp hack/git-verify-wrapper.sh "${{targets.subpkgdir}}"/usr/bin/git-verify-wrapper.sh
+
+  - name: argo-cd-compat
```
Comment (mattmoor) [inline argo-cd-2.7.yaml:92]: `${{package.name}}-compat`

File: argo-cd-2.7.yaml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-cmp-server
+      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-application-controller
+      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-notifications
+      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-applicationset-controller
+      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-k8s-auth
+
+  - uses: strip
+
+subpackages:
+  - name: argo-cd-repo-server
```
Comment (mattmoor) [inline argo-cd-2.7.yaml:70]: as below `${{package.name}}-repo-server`

File: argo-cd-2.7.yaml (hunk 2 above)
Comment (tcnghia) [reply argo-cd-2.7.yaml:92]: No, package.name is argo-cd-${version-stream} now

File: argo-cd-2.8.yaml (hunk 1 above)
Comment (tcnghia) [reply argo-cd-2.8.yaml:108]: Done. Thanks.

File: argo-cd-2.7.yaml (hunk 3 above)
Comment (tcnghia) [reply argo-cd-2.7.yaml:70]: No, package.name is argo-cd-${version-stream} now

**Review by mattmoor — APPROVED** [review]

**Pull request merged**